docs/wiki/Home.md

```diff
@@ -59,7 +59,7 @@ If you're unfamiliar with Infrastructure as Code, or wonder how you can use the
   - [Pipeline usage](./The%20CI%20environment%20-%20Pipeline%20usage)
 - [Interoperability](./Interoperability)
   - [Bicep to ARM/JSON conversion](./Interoperability%20-%20Bicep%20to%20ARM%20conversion)
-  - [Register ADO pipelines](./Interoperability%20-%20Register%20ADO%20pipelines)
+  - [Transition to Azure DevOps Pipelines](./Interoperability%20-%20Transition%20to%20Azure%20DevOps%20Pipelines)
```

Comment:
Why 'Transition to'? The ADO pipelines are already there, so a user would just need to register them.

Reply:
Comes from the lab. I'll revert it back to Register

Reply:
dan